### PR TITLE
[QuickOpen] show the end of the path

### LINKED ISSUE #8124: [QuickOpen] we should show the end of the path
I've found it can be hard to see the relevant paths the way we're truncating

Screen Shot 2019-03-13 at 5 31 26 PM

### PR DESCRIPTION
**Pull request by th7nder**
Fixes #8124 
Implemented a version with showing only the end of the path.
I am not sure if we need ellipsis in the middle, because in most cases we care about last directories.

### Summary of Changes

* removed truncation in JS 
* added an ellipsis on the start of the line

### CODE REVIEW

File: src/components/shared/ResultList.css [hunk 1]
```diff
@@ -113,10 +113,12 @@
   word-break: break-all;
   /** https://searchfox.org/mozilla-central/source/devtools/client/themes/variables.css **/
   color: var(--grey-40);
-  margin-left: 15px;
+  margin-left: 5px;
```
Comment (WenInCode) [inline src/components/shared/ResultList.css]: Just curious what your motivation is for changing this margin? Is it just adding more space for the url? I'm not sure if we have a guideline for spacing in this content. I think the larger margin makes it a bit more readable, but I'd love to hear your thoughts on it 😄

Comment (th7nder) [reply src/components/shared/ResultList.css]: It felt like there is too much space when I added an ellipsis at the beginning, but now I am not so sure :D good point!
image

Comment (darkwing) [reply src/components/shared/ResultList.css]: I think we should keep left and right spacing consistent; IMO this change makes it too close to the right edge.

File: src/components/shared/ResultList.css [hunk 2]
```diff
@@ -113,10 +113,12 @@
...
   text-overflow: ellipsis;
   overflow: hidden;
   white-space: nowrap;
+  direction: rtl; /* make ellipsis on the beginning of a string */
```
Comment (darkwing) [inline src/components/shared/ResultList.css]: This is a solid trick but I'm wondering if @yzen could weigh in on accessibility with it.

File: src/components/shared/ResultList.css (hunk 1 above)
Comment (th7nder) [reply src/components/shared/ResultList.css]: Sure, I'll change it back.

File: src/components/shared/ResultList.css (hunk 2 above)
Comment (yzen) [reply src/components/shared/ResultList.css]: It seems to have no negative effect on screen readers. I think this should be fine.

**Pull request merged**